ui/open-api-ui/src/test/java/io/smallrye/openapi/ui/IndexCreatorTest.java

```diff
@@ -64,15 +67,16 @@ void testCreateWithStringBooleanOption() throws IOException {
 
         String s = new String(indexHtml);
 
-        assertTrue(s.contains("<title>SmallRye OpenAPI UI</title>"));
-        assertTrue(s.contains("<link rel=\"stylesheet\" type=\"text/css\" href=\"theme-feeling-blue.css\" >"));
-        assertTrue(s.contains("<link rel=\"stylesheet\" type=\"text/css\" href=\"style.css\" >"));
-        assertTrue(s.contains("url: '/openapi',"));
-        assertTrue(s.contains("<img src='logo.png' alt='SmallRye OpenAPI UI'"));
-        assertTrue(s.contains("dom_id: '#swagger-ui',"));
-        assertTrue(s.contains("deepLinking: true,"));
-        assertTrue(s.contains("filter: 'bla',"));
-        assertTrue(s.contains("syntaxHighlight: false,"));
+        assertThat(s).contains("<title>SmallRye OpenAPI UI</title>");
+        assertThat(s).contains("<link rel=\"stylesheet\" type=\"text/css\" href=\"theme-feeling-blue.css\" >");
+        assertThat(s).contains("<link rel=\"stylesheet\" type=\"text/css\" href=\"style.css\" >");
+        assertThat(s).contains("url: '/openapi',");
+        assertThat(s).contains("<img src='logo.png' alt='SmallRye OpenAPI UI'");
+        assertThat(s).contains("dom_id: '#swagger-ui',");
+        assertThat(s).contains("deepLinking: true,");
+        assertThat(s).contains("filter: 'bla',");
+        assertThat(s).contains("syntaxHighlight: false,");
+
```

Comment:
This is really no change ? Right ?

Reply:
No changes in the code the only changes are  assertTrue to assertThat to show the error more friendly, Should I remove or its ok?

Reply:
Remove for this PR. If you feel strongly about it create a new PR/discussion and we can discuss there